share_app/src/main/java/org/odk/share/fragments/FilledFormsFragment.java

```diff
@@ -90,16 +90,19 @@ public void onResume() {
 
     @Override
     public void onLoadFinished(@NonNull Loader<Cursor> loader, Cursor cursor) {
-        cursor.moveToFirst();
-        instanceAdapter = new InstanceAdapter(getActivity(), cursor, this::onListItemClick, selectedInstances);
-        recyclerView.setAdapter(instanceAdapter);
-        setEmptyViewVisibility(cursor.getCount());
-        if (instanceAdapter.getItemCount() > 0) {
-            toggleButton.setText(getString(R.string.select_all));
-            toggleButton.setEnabled(true);
-        } else {
-            toggleButton.setEnabled(false);
-        }
+        if (cursor != null) {
+            cursor.moveToFirst();
+            instanceAdapter = new InstanceAdapter(getActivity(), cursor, this::onListItemClick, selectedInstances);
+            recyclerView.setAdapter(instanceAdapter);
+            setEmptyViewVisibility(cursor.getCount());
+            if (instanceAdapter.getItemCount() > 0) {
+                toggleButton.setText(getString(R.string.select_all));
+                toggleButton.setEnabled(true);
+            } else {
+                toggleButton.setEnabled(false);
+            }
+        } else
+            setEmptyViewVisibility(0);
```

Comment:
Make sure to write every block in braces `{}` even if the block contains a single line statement.